test/Microsoft.ML.Predictor.Tests/TestPredictors.cs

```diff
@@ -146,14 +146,17 @@ public void EarlyStoppingTest()
         /// <summary>
         /// Multiclass Logistic Regression test.
         /// </summary>
-        [LessThanNetCore30OrNotNetCoreFact("netcoreapp3.0 output differs from Baseline")]
+        //[LessThanNetCore30OrNotNetCoreFact("netcoreapp3.0 output differs from Baseline")]
+        [Theory, VaryingTolerance(4)]
         [TestCategory("Multiclass")]
         [TestCategory("Logistic Regression")]
         //Skipping test temporarily. This test will be re-enabled once the cause of failures has been determined
-        [Trait("Category", "SkipInCI")]
-        public void MulticlassLRTest()
+        public void MulticlassLRTest(int tolerance)
         {
-            RunOneAllTests(TestLearners.multiclassLogisticRegression, TestDatasets.iris, digitsOfPrecision: 4);
+            // Replacing LessThanNetCore30OrNotNetCoreFactAttribute
+            if (AppDomain.CurrentDomain.GetData("FX_PRODUCT_VERSION") != null)
+                return;
```

Comment:
Same here too. Please try to preserve the log message in the Fact that you replaced.